File: two-sum/two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        for index in range(len(nums)):
            for val in range(index+1,len(nums)):
                if target == nums[index]+nums[val]:
                    return[index,val]

File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {

        Map<Integer, Integer> mp = new HashMap<>();

        for(int i = 0; i < nums.length; i++) {
            int t1 = target - nums[i];

            if(mp.containsKey(t1)) {
                return new int[]{mp.get(t1), i};
            }
            mp.put(nums[i], i);
        }

        return new int[0];
        
    }
}